kube-state-metrics/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG UPSTREAM_VERSION
+ARG UBI_IMAGE=registry.access.redhat.com/ubi8/ubi:8.6
+
+## Use original image to copy files from
+## ref: https://github.com/kubernetes/kube-state-metrics/blob/v${UPSTREAM_VERSION}/Dockerfile
+FROM public.ecr.aws/sumologic/kube-state-metrics:v${UPSTREAM_VERSION} as builder
+
+## Build RedHat compliant image
+FROM ${UBI_IMAGE}
+ARG UPSTREAM_VERSION
```

Comment:
In the other PR you removed `UBI_IMAGE` var, shouldn't you do the same here?

Reply:
the other one bases on this one :D